backend/api/Controllers/UserController.cs

```diff
@@ -32,11 +39,13 @@ public class UserController : ControllerBase
         /// <param name="logger"></param>
         /// <param name="configuration"></param>
         /// <param name="requestClient"></param>
```

Comment:
Missing comment for parameters.  For all the value they bring us currently :P  One day...

Reply:
I'll fix this.